src/SonarScanner.MSBuild/BootstrapperClass.cs

```diff
@@ -164,8 +167,15 @@ private int PostProcess()
             }
             else
             {
-                var postProcessor = this.processorFactory.CreatePostProcessor();
-                succeeded = postProcessor.Execute(this.bootstrapSettings.ChildCmdLineArgs.ToArray(), config, teamBuildSettings);
+#if NETCOREAPP2_1
+
+                WarningsSerializer.Serialize(
```

Comment:
The easiest way to test this is probably through IT, where you'd start by looking at `TestUtils.newScanner`, modifying the scanner location